src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,44 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
-    private final StorageDao storageDao = new StorageDaoImpl();
+    private static final StorageDao storageDao = new StorageDaoImpl();
+    private static final int MIN_USER_AGE = 18;
+    private static final int MIN_PASSWORD_LENGTH = 6;
 
     @Override
     public User register(User user) {
-        return null;
+        checkAge(user);
+        checkPassword(user);
+        checkLogin(user);
+        storageDao.add(user);
+        return user;
+    }
+
+    public static void checkPassword(User user) {
```

Comment:
```suggestion
    private static void checkPassword(User user) {
```
check this for all methods